app/models/user.rb

```diff
@@ -6,6 +6,11 @@ class User < ApplicationRecord
   validates :user_name, uniqueness: true, presence: true
```

Comment:
user_name no longer needs to be unique. We will enforce uniqueness on the uid as given by google oauth